src/components/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default function ({ title, size, addClasses, type, isActive }) {
+    let typeClasses = {
```

Comment:
Looks like a constant. You can move this code out of the function:
```
let typeClasses = {
            outline: 'btn-outline-primary',
            secondary: 'btn-secondary'
        },
        sizeClasses = {
            small: 'btn-sm'
      };
```

Reply:
Done